Documentation/Signoffs/AbigailKennedy-Signoff-MotorControl.md

```diff
@@ -0,0 +1,101 @@
+# Motor Control Subsystem
+## Function of the Subsystem
+The motor control subsystem takes serial signals from the master control and determines the signals to send to the motor drivers to move the robot. To be successful, the robot must be able to move with a maximum speed of 2 ft/s, move up an incline of 25 degrees, turn 360 degrees, and move forward and backward. 
+
+## Constraints
+- C9: The robot shall travel inclines and declines up to 25 degrees. 
+- C10: The robot shall turn 360 degrees left and right and move forwards and backward based on sensor inputs.
+- C13: The robot shall possess a maximum speed of 2 feet per second.
+- The robot's speed will be calculated assuming the robot weighs 20 pounds. This is done due to the maximum weight limit for competition.
+
+### Socioeconomic Consideration
+- The motors chosen should operate at about 70% of the maximum efficiency to ensure power can be conserved and to reduce charging costs. Additionally, this will give the robot enough extra power when traversing inclines. 
+- C14: The robot shall have a user manual that explains all functions and capabilities of the robot. 
+        - The work done on this design and the implementation will be documented thoroughly to ensure the user manual is as helpful as possible for future groups using the robot.
+
+## Buildable Schematics
+
+### Electrical Schematic
+![image](https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/main/Documentation/Images/Motor-Control/MotorControl-Schematic.png)
+
+This electrical schematic shows the connections between the microcontroller, motor drivers, and motors. 
+
+### Motor and Wheel Assembly
```

Comment:
There is a disconnection between this assembly part and chasis. Are you planning to have more detailed description in chasis design or is it in attachment signoff? I only found a motor attachment in that signoff.

Reply:
I agree, and I added another picture and more explanation. The motor attachment in that signoff will connect to the motor mount I explained here.

Reply:
I see two motors connecting to one motor driver. The current drawing from the motor driver, 0.72A, I assume is from the driver to one motor. Overall consumes 0.72*2 A?

Reply:
It is also suggested to have each electrical component's spec attached to the signoff with appropriate numbering in this section as references.

Reply:
I added the references, which I definitely should've had before. 

I figured it would be fine for the driver output current since the datasheet says the maximum is 2A per channel, and I reworded it in the signoff for more clarity. However, I'm unsure since the stall current is much higher (5.5 A). Since the maximum torque needed by a motor to run the worst-case scenario is maximum efficiency and 0.72A, I plan to kill the motors if they start to reach the 2A driver limit. Is this an acceptable approach, or should I look for another way to go about it?